src/main/java/entity_layer/CommonTaskTree.java

```diff
@@ -30,7 +68,12 @@ public void addSubTaskTree(TaskTree taskTree) {
      */
     @Override
     public boolean removeSubTaskTree(TaskTree taskTree) {
-        return false;
+        if (this.subTaskTrees.contains(taskTree)) {
+            this.subTaskTrees.remove(taskTree);
```

Comment:
I think this could cause issues for completion percentages because after a subtask tree is removed, there is no update on the completion percentages; your test cases should show you this

Reply:
I forgot to do implement this! Good catch, I'll write a case for it now, too.